index.html

```diff
@@ -1164,6 +1164,53 @@ <h3>Content Distribution Networks</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Decoy Values</h3>
+
+      <p>
+The use of decoy values in status lists by <a>issuers</a> has been explored as
+a mechanism to increase the privacy of <a>subjects</a> by further hiding the
+size of the group associated with a status list as well as when values in the
+list change. While algorithms for employing decoy values are out of scope for
+this specification, implementers are advised to heed the guidance in this
+section when considering the use of decoy values.
+      </p>
+      <p>
+Decoys might help privacy in some cases, but harm privacy in others. For
+example, if status list entry indexes are allocated in a purely random fashion,
+adding decoys harms privacy because it reduces the group privacy size by the
+number of decoys added to the group. A purely random allocation of indexes
+ensures that decoys are not necessary unless a bulk operation, such as setting
+every credential issued in a list to the same status (such as "revoked"),
+would reveal the true size of the group.
```

Comment:
Tweaked some language here, but I'm still having trouble finding the value of decoys in the "bulk update" case (again, presuming random allocation of indexes is already used).

I don't think I fully understand the threat model. I understand that it could be problematic for an adversary to learn the "true group size" when it is significantly smaller than the total size of the list. Learning this would allow an adversary to make more accurate guesses about the use of a particular VC if they also:

1. knew of at least one VC associated with the list, and
2. could observe retrieval of the list by other parties.

Is there a problem if an adversary learns the "true group size" is close to or the same size as the total size of the list (for a list with over ~100K indexes)? Is it a goal for an adversary to simply never learn how many total VCs have been issued -- and, if so, is this still about subject privacy or is it about issuer privacy? What if they learn N out of M have been issued? Is that ok -- where is the line? Is "the total number" special for some reason?

Further analyzing this "bulk update" case, there is of course a presumption that the adversary does not already know the "true group size" (since this is the threat). And my understanding is that a "bulk update" means that an adversary could observe N-many bits flip at once (for N > 1).

Now supposing that an adversary also knows that there are no decoys, how would simply observing any number of N bits change (where N is less than the total list size) tell them the "true group size"? How are they to know whether a "bulk update" changed all of the statuses of the "true group size" and not just some subset of it?

Is there an additional presumption that they also know that, for example, all VCs in that particular list (or perhaps, all VCs for that issuer across all lists) are about to be revoked -- and they are just waiting to see how many bits flip? Why would an adversary presume there is a single list or that they have knowledge of (and an address for) all current lists such that they could observe them all?

For this threat to be realized, it seems there may be a considerable amount of information that the adversary needs to have -- all whilst not being able to make a good guess at the "true group size" using that very same information.

```suggestion
number of decoys added to the group. A random allocation of indexes already
hides the true group size, ensuring that decoys are not necessary unless a bulk
operation, such as setting every credential issued in a list to the same status
(such as "revoked"), would reveal the true size of the group to an adversary
that also knew that the bulk update was to be applied to all non-decoy indexes.
```